app/boards/shields/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2020 Pete Johanson
```

Comment:
We're still working on fixing up existing boards, but please use the "single author file" version like so:

```suggestion
# Copyright (c) 2020 The ZMK Contributors
```

Reply:
added
# Copyright (c) 2020 The ZMK Contributors
# SPDX-License-Identifier: MIT